src/samples/AmazonSimpleQueueService/README.md

```diff
@@ -28,4 +28,4 @@ The basic steps for running the Amazon SQS sample are:
 
 **NOTE:** The sample also includes an Ant build.xml file to run the sample.
 
-See the [Amazon SQS Getting Started Guide](http://docs.amazonwebservices.com/AWSSimpleQueueService/latest/SQSGettingStartedGuide/) for step-by-step instructions on running this sample.
+See the [tutorials](http://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-tutorials.html) in the _Amazon SQS Developer Guide_ for step-by-step description of this sample.
```

Comment:
The tutorials link is not a step-by-step description of this sample. May be we should change the link to "See the Amazon SQS Tutorials for more samples."